cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -772,16 +771,17 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         }
 
         // Try to retrieve the code and location from the parent node, if it is a base stmt
-        var baseStmt = parentNode as? Python.AST.BaseStmt
-        return if (baseStmt != null) {
-            result.codeAndLocationFromChildren(baseStmt)
-        } else {
-            // Otherwise, continue without setting the location
-            log.warn(
-                "Could not set location on wrapped block because the parent node is not a python statement"
-            )
-            result
+        val ast = parentNode as? Python.AST.AST
+        if (ast != null) {
```

Comment:
Why are we removing this warning? Is it irrelevant?

Reply:
Yes it is, this is needed because of the stupid workaround with interfaces vs. abstract classes, all nodes are ` Python.AST.AST` but we need to use the interface because of the async workaround.